tools/ci/branch_cutover_and_rebase.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+SPLUNK_MAJOR_VERSION="1.13"
```

Comment:
I was thinking this would be beneficial as we need not monitor for a minor release version and scheduling takes care of the upgrade. This variable doesn't set anything. It's just a reference variable for filtering.

Reply:
Yeah what I mean is right now it requires a manual update to know what our current major version is and what we should be looking for. So we'd need a manual update to start looking for 1.14.x versions for example. If we took autoamtically from the default branch name instead then that sort of naturally flows from us changing the default branch in the UI every time we make an upgrade

Reply:
Oh! I get it. I have updated the script to make use of default branch to get the major version and current splunk release version to be default.